modin/engines/ray/generic/io.py

```diff
@@ -32,11 +32,13 @@ def file_exists(file_path):
             from botocore.exceptions import NoCredentialsError
 
             s3fs = S3FS.S3FileSystem(anon=False)
+            exists = False
```

Comment:
When travis runs this code, it gets `False` when checking without credentials, but `True` when checking with credentials, so I changed it so that it checks both with and without credentials, and returns `True` if it exists in either. Basically, this is what is happening rn:
```python
s3fs = S3FS.S3FileSystem(anon=False)
try:
    return s3fs.exists(file_path) # Returns false, causing modin to default to pandas backend
catch NoCredentialsError: # Not thrown, but it looks like it maybe would've been thrown in the past? This not being thrown is probably the error.
    s3fs = S3FS.S3FileSystem(anon=True)
    return s3fs.exists(file_path) # Would return true, allowing test to continue as planned
```